bin/replication.js

```diff
@@ -105,6 +110,72 @@ program.version(version);
         });
 });
 
+program
+    .command('multi-site-setup')
+    .description('if a Zenko site is included as a replication endpoint for a' +
+        ' one-to-many setup, "target-bucket" and "target-profile" are needed')
+    .option('--source-bucket <name>', '[required] source bucket name')
+    .option('--source-profile <name>',
+            '[required] source aws/credentials profile')
+    .option('--target-bucket <name>', '[optional] target bucket name ' +
+            '(necessary if adding a Zenko site as a replication endpoint)')
+    .option('--target-profile <name>', '[optional] target aws/credentials ' +
+            'profile (necessary if adding a Zenko site as a replication ' +
+            'endpoint)')
+    .option('--multi-sites <site1,site2,...>', '[required] comma separated ' +
+            'list of sites')
+    .action(options => {
+        const log = new Logger('BackbeatSetup').newRequestLogger();
+        const sourceBucket = options.sourceBucket;
+        const sourceProfile = options.sourceProfile;
+        let targetBucket = options.targetBucket;
+        const targetProfile = options.targetProfile;
+        const multiSites = options.multiSites;
+
+        const destinationEndpoint = config.getBootstrapList()
+            .find(dest => Array.isArray(dest.servers));
+
+        const hasIncorrectMultiSites = !multiSites || multiSites.split(',')
+            .some(site => site.length === 0);
+        const hasValidOptionals = (targetBucket || targetProfile) ?
```

Comment:
Suggestions to improve clarity:

- avoid the use of `?:` operator: when the check gets complex enough, prefer a regular `if..else` construct

- logically speaking, i.e. when used in a boolean context, I think `!(!targetBucket || !targetProfile)` is equivalent to `targetBucket && targetProfile`

- avoid double negation when checking: change var to `hasCorrectMultiSites`